docs/source/algorithms/SetSample-v1.rst

```diff
@@ -82,11 +93,13 @@ be in centimeters):
 - ``FlatPlate``: Width, Height, Thick, Center, Angle
 - ``Cylinder``: Height, Radius, Center
 - ``HollowCylinder``: Height, InnerRadius, OuterRadius, Center
-- ``CSG``: Value is a string containing any generic shape as detailed in
-  :ref:`HowToDefineGeometricShape`
+- ``FlatPlateHolder``: Width, Height, Thick, Center, Angle, FrontThick, BackThick. This is a ``CSG`` union of 2 ``FlatPlate``s tightly wrapping a ``FlatPlate`` sample.
+- ``HollowCylinderHolder``: Height, InnerRadius, InnerOuterRadius, OuterInnerRadius, OuterRadius, Center. This is a ``CSG`` union of 2 ``HollowCylinder``s tightly wrapping a ``HollowCylinder`` sample.
```

Comment:
There are a couple of problems with the markup in the FlatPlateHolder and HollowCylinderHolder sections (yellow highlights are mine):

![SetSampleDocScreenshot](https://user-images.githubusercontent.com/56295817/83733332-4e9ba980-a645-11ea-8648-52831b2fa4c2.PNG)